mappings/net/minecraft/world/gen/DeepslateBlockSource.mapping

```diff
@@ -3,8 +3,11 @@ CLASS net/minecraft/class_5842 net/minecraft/world/gen/DeepslateBlockSource
 	FIELD field_28910 seed J
 	FIELD field_28911 defaultBlock Lnet/minecraft/class_2680;
 	FIELD field_28912 deepslateState Lnet/minecraft/class_2680;
+	FIELD field_31468 DEFAULT_MIN_Y I
+	FIELD field_31469 DEEPSLATE_MAX_Y I
```

Comment:
Don't think the `DEEPSLATE_` prefix is necessary unless... 
there is a protected/public static field with same name in superclass/interfaces that this class or its subclasses call with getstatic with this class/subclass as owner
```suggestion
	FIELD field_31469 MAX_Y I
```